composer.json

```
@@ -10,12 +10,12 @@
   "prefer-stable": true,
   "minimum-stability": "dev",
   "require": {
-    "php": ">=8.1 <8.4",
+    "php": ">=8.1",
```

It is great to include a limit version. 

```suggestion
    "php": ">=8.1 <8.5",
```